### DISCUSSION TITLE
Remove `null` from `onChange` if `isClearable={false}` is passed in.

### DISCUSSION THREAD

**Discussion by Nokel81** [JedWatson/react-select#5151] [Ideas]
It would be great if I could set `isClearable={false}` and then not have to deal with the `null` case in my `onChange` handler. Could this be possible?

**Comment by Methuselah96** [comment]
Currently not supported, but I agree that would be nice. Feel free to create a pull request.

**Reply by Nokel81** [reply]
Cool, will try and find some time to make a PR then.

**Comment by j-perl** [comment]
Hi, has this ever been implemented ? I was thinking the same. Would gladly create a pull request if not and if null was not intended in this case. @Methuselah96
